### PR TITLE
[VuePress] Change-log pages

### PR DESCRIPTION
**Pull request by bretterer**
This PR updates the Changelog to be dynamic based on the files in the `change-log` folder.  This is an expirement to see if we like it better, or different than running the `yarn changelog` command

Doing this makes it so live reload works when working with the changelog files.

**Adds**
- Lodash as a dependency to deal with sorting of changelog

**Change**
- Makes it so the changelog is more dynamic and does not require a script. 
- Moves everything from devDep's to Dep's in package.json
- Updates Sidebar to the correct html format

### CODE REVIEW

File: docs/.vuepress/theme/layouts/ChangeLog.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+</template>
+
+<script>
+import _ from "lodash"
+export default {
+  name: 'ChangeLog',
+  computed: {
+    changelogPages: function () {
+      let pages = this.$site.pages
+        .filter(pages => pages.path.startsWith("/change-log/2"))
```
Comment (bdemers) [inline docs/.vuepress/theme/layouts/ChangeLog.vue]: I'm guessing this is to filter out any non-date based change logs?

File: docs/.vuepress/theme/layouts/ChangeLog.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+      return _.orderBy(pages, 'path').reverse()
+    }
+  },
+  data: function () {
+    return {
+      site: {
+        contact_support_lc: '<a href="https://support.okta.com/help/open_case?_ga=2.35087137.18686933.1540307948-1734068328.1494253825" target="_blank">Support</a>'
```
Comment (bdemers) [inline docs/.vuepress/theme/layouts/ChangeLog.vue]: That is a great URL :trollface:

File: docs/.vuepress/theme/layouts/ChangeLog.vue (hunk 1 above)
Comment (bretterer) [reply docs/.vuepress/theme/layouts/ChangeLog.vue]: yes, that and to not target the index.md file

File: docs/.vuepress/theme/layouts/ChangeLog.vue (hunk 2 above)
Comment (bretterer) [reply docs/.vuepress/theme/layouts/ChangeLog.vue]: This is going to be removed

**Pull request merged**